hw/ip/i2c/data/i2c_testplan.hjson

```diff
@@ -445,4 +445,144 @@
+            Cover the FIFO reset bits in FIFO_CTL register
+            Cross fmt_threshold with FMTRST
+            Cross rx_threshold with RXRST
+            Cross fmt_overflow with FMTRST
+            Cross rx_overflow with RXRST
+            Cross acq_overflow with ACQRST
+            Cross tx_overflow with TXRST
+            '''
+    }
+    {
+      name: i2c_fifo_level_cg
+      desc: '''
+             Cover the trigger level for FMT_FIFO and RX_FIFO interupts
+             * Cross fmt_threshold interrupt with FMT_FIFO trigger level
+             * Cross rx_threshold interrupt with RX_FIFO trigger level
+             Cover the FIFO levels in FIFO_STATUS
+            '''
+    }
+    {
+      name: i2c_operating_mode_cg
+      desc: '''
+            * Cover the operating mode( Host/Target) of DUT
+            * Cover the operating mode( Host/Target) of TB
+            * Cross the operating modes of DUT and TB
+            * Cover the SCL frequency
+            * Cross the SCL frequency with operating mode of DUT
+            '''
+    }
+    {
+      name: i2c_rd_wr_cg
+      desc: '''
+            Cover the number of bytes read from Target
+            Cover the number of bytes written to Target
+            * Cross the number of bytes read in Target and Host mode of DUT
+            * Cross the number of bytes written in Target and Host mode of DUT
+            Cover the address match feature of IP
+            * Cross the address sent or received with Host and Target mode of DUT
+            * Cross write address with address match
+            * Cross read address with address match
+            Cover values in Read bytes
+            * Cross with operating mode of I2C
+            Cover values in Write bytes
+            * Cross with operating mode of I2C
+            '''
+    }
+    {
+      name: i2c_status_cg
+      desc: '''
+            Cover the fields in i2c.STATUS register
+            '''
+    }
+    {
+      name: i2c_scl_sca_override_cg
+      desc: '''
+            Cover TX override enable bit TXOVRDEN
+            Cover override value of SCL
+            Cover override value of SDA
+            Cover oversampled value of SCL
+            Cover oversampled value of SDA
+            Cross override vaue of SCL with override enable
+            Cross override vaue of SDA with override enable
+            '''
+    }
+    {
+      name: i2c_cmd_complete_cg
+      desc: '''
+            Cover cmd_complete interrupt
+            Cross cmd_complete with Host mode Read and Write operation
+            Cross cmd_complete with Target mode Read and Write operation
+            '''
+    }
+    {
+      name: i2c_fmt_fifo_cg
+      desc: '''
+            Cover the values supported by FDATA register
+            * Data byte (all flags set to zero)
+            * Write address byte
+            * Read address byte
+            * Read a number of bytes from Target with NACK
+            ** Cover values ranging from 1-128 Bytes
```

Comment:
Is double asterisk a valid syntax here? Maybe it's better to just indent here?

Reply:
Updated as per comments